src/models/sensor-gdx-manager.ts

```diff
@@ -123,10 +118,38 @@ export class SensorGDXManager extends SensorManager {
       return this.hasData;
     }
 
+    // This value gets sent to gdxDevice.start() to set the measurement period.
+    // It gets used when we've actually started the sensor data graphing.
+    getMeasurementPeriod() {
+      // The Heart Rate sensor is only able to collect 1 sample per second.
+      // Currently HEARTBEAT_INTERVAL_MS is set to 1000ms (1 second).
+      // If that changes we'll need to manually return 1000 here.
+      if (this.gdxDevice.orderCode === "GDX-EKG") {
+        return HEARTBEAT_INTERVAL_MS;
+      } else {
+        return this.gdxDevice.minMeasurementPeriod;
+      }
+    }
+
+    // This value also gets sent to gdxDevice.start() to set the measurement period.
+    // It gets used when we're displaying the calibration values, not during collecting / graphing.
+    getHeartbeatInterval() {
+      if (this.gdxDevice.orderCode === "GDX-EKG") {
+        // See notes above in getMeasurementPeriod().
+        return HEARTBEAT_INTERVAL_MS;
```

Comment:
Same comment as above, I think this should change to the new `MIN_EKG_INTERVAL_MS` constant.